spaces/models.py

```diff
@@ -3,7 +3,45 @@
 
 
 class District(models.Model):
+    TYPE_MAJOR_DISTRICT = "major_district"
+    TYPE_DISTRICT = "district"
+    TYPE_SUB_DISTRICT = "sub_district"
+
+    TYPE_CHOICES = (
```

Comment:
Just curious if these are used as way to restrict the parent child hierarchy? Like only major district can be parent of district and so on.

Reply:
This is pretty much fixed hierarchy in district system of Helsinki. Validation you described would represent the real world scenario and could be implemented but its probably not mandatory feature because districts are probably created once (programmatically maybe) and then only changed if changes happens like once in 10 years or something.

Reply:
Yeah, didn't mean this would necessarily need validation at least in the current PR